traffic_ops/traffic_ops_golang/crconfig/config.go

```diff
@@ -0,0 +1,69 @@
+package crconfig
```

Comment:
missing Apache license header on several of the files here...

Reply:
Fixed